### PR TITLE
Remove float property to let the margin between events appear on bubble message layout

### PR DESCRIPTION
**Pull request by luixxiul**
Fixes https://github.com/vector-im/element-web/issues/21861

From:
before

To:
after

Steps to confirm the fix:

1. Set the layout to Message bubble
2. Send something on a room
3. Send something again
4. Delete both messages

type: defect

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Remove float property to let the margin between events appear on bubble message layout ([\#8373](https://github.com/matrix-org/matrix-react-sdk/pull/8373)). Fixes vector-im/element-web#21861. Contributed by @luixxiul.

Preview: https://pr8373--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/rooms/_EventBubbleTile.scss [hunk 1]
```diff
@@ -593,6 +593,13 @@ limitations under the License.
             right: 0; // match alignment of the hover background to that of chat bubbles
         }
     }
+
+    // style a list inside the generic event list
+    .mx_GenericEventListSummary_unstyledList {
+        display: flex;
+        flex-direction: column;
+        margin: 0;
```
Comment (robintown) [inline res/css/views/rooms/_EventBubbleTile.scss:601]: This `margin: 0;` doesn't seem to be necessary

**Review by robintown — COMMENTED** [review]
LGTM otherwise

**Comment by luixxiul** [comment]
In any cases `mx_EventTile_msgOption` has only `mx_EventTile_readAvatars` as a child element, and elements of `mx_EventTile_readAvatars` is placed with `position: absolute`, which makes `float: right` redundant, so removing the property should make more sense than adding rules, unless something would be expected to be displayed on the left of the bubbles (ie. in the yellow area below).

after
after1

Purple: padding
Yellow: margin

Fixing the border-radius is out of scope of this PR.

**Review by robintown — APPROVED** [review]
The float change is potentially dangerous, so we'll want to watch out for regressions, but in my testing I couldn't see anything wrong. Thank you!

**Comment by luixxiul** [comment]
I'll keep an eye on regressions as well, thanks!

**Pull request merged**